skimage/measure/_structural_similarity.py

```diff
@@ -3,36 +3,137 @@
 __all__ = ['structural_similarity']
 
 import numpy as np
+from scipy.ndimage.filters import uniform_filter, convolve1d
 
 from ..util.dtype import dtype_range
-from ..util.shape import view_as_windows
 
 
-def structural_similarity(X, Y, win_size=7,
-                          gradient=False, dynamic_range=None):
+def gaussian_filter2(X, sigma=1.5, size=11):
+    """ nD Gaussian filter with specific window extent.
+
+    matches the implementation of Wang. et. al.
+
+    Parameters
+    ----------
+    X : ndarray
+        image
+    sigma : float
+        Gaussian standard deviation (pixels)
+    size : float
+        Gaussian kernel extent (pixels)
+
+    Returns
+    -------
+    X : ndarray
+        filtered image
+
+    Notes
+    -----
+    scipy.ndimage.gaussian is very similar, but uses a 13 tap FIR filter
+    rather than the 11 tap one of Wang. et. al.
```

Comment:
It is excellent to have this comparison.  I am happy to use the 13-tap
filter, especially now we have the confidence that the results are
backed by another implementation.